### DISCUSSION TITLE
French kissing

### DISCUSSION THREAD

**Discussion by Dwfreegethub** [littlesera/LSCG#328] [Ideas]
I have always found that a French kiss was an automatic thing that anyone could French kiss anyone at any time. Any chance to put French kisses behind a allowed or denied menu..

 On the same idea if you declined the French kiss the one kissing you could have a force option. Could possibly use most of the same code as force drinks.

Also possible would be nice to have a white list or black list along with this.

**Comment by littlesera** [comment]
So the idea is an additional, more detailed layer of permissions on this activity beyond the regular BC block/allow/arousal settings?

**Comment by Dwfreegethub** [comment]
I was not even aware/forgot I could completely block actions that way.. But
I was thinking more another layer. So if a friend walks in I want them to
kiss me. If a random stranger walks in I want them to know that I don't
want to be kissed by them and if they choose to try and force a kiss well
that becomes rp.

The way I pictured it in my head is when someone tries to French kiss you a
menu (with a timer) pops up with at least the options of allowing or
declining the kiss. Additional options of white-list or blacklist might be
useful as well.  Then if you decline (and the person is using LSCG) they
would get something saying that you declined the kiss and an option to
force the kiss.

You always make any of the ideas I suggest way better so I can see you
improving my thoughts.. Assuming this is even doable.

Each option could create custom text obviously.
